examples/01-filter/image-fft-perlin-noise.py

```diff
@@ -0,0 +1,206 @@
+"""
+.. _image_fft_perlin_example:
+
+Fast Fourier Transform with Perlin Noise
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This example shows how to apply a Fast Fourier Transform (FFT) to a
+:class:`pyvista.UniformGrid` using :func:`pyvista.UniformGridFilters.fft`
+filter.
+
+Here, we demonstrate FFT usage by first generating Perlin noise using
+:func:`pyvista.sample_function() <pyvista.core.imaging.sample_function>` to
+sample :func:`pyvista.perlin_noise <pyvista.core.common_data.perlin_noise>`,
+and then performing FFT of the sampled noise to show the frequency content of
+that noise.
+"""
+
+import numpy as np
+
+import pyvista as pv
+
+###############################################################################
+# Generate Perlin Noise
+# ~~~~~~~~~~~~~~~~~~~~~
+# Start by generating some `Perlin Noise
+# <https://en.wikipedia.org/wiki/Perlin_noise>`_ as in
+# :ref:`perlin_noise_2d_example` example.
+#
+# Note that we are generating it in a flat plane and using a frequency of 10 in
+# the x direction and 5 in the y direction. Units of the frequency are
+# ``1/pixel``.
```

Comment:
There's an inherent mismatch in pluralization here: "units" (plural) vs "frequency" (singular) vs "1/pixel" (singular-ish?). I'm not happy with either "are" nor "is" here :D